src/tasks/spelldamage.ts

```diff
@@ -151,6 +157,21 @@ export const SpellDamageQuest: Quest = {
       outfit: { modifier: "HP 500max, Spooky Resistance", familiar: $familiar`Exotic Parrot` },
       limit: { tries: 1 },
     },
+    {
+      name: "Stick-Knife Trick",
+      ready: () =>
+        get("instant_stickKnifeOutfit") !== "" &&
+        myClass() === $class`Pastamancer` &&
+        have($item`Stick-Knife of Loathing`) &&
+        (have($skill`Bind Undead Elbow Macaroni`) || myThrall() === $thrall`Undead Elbow Macaroni`),
+      completed: () => haveEquipped($item`Stick-Knife of Loathing`),
+      do: (): void => {
+        if (myThrall() !== $thrall`Undead Elbow Macaroni`)
+          useSkill($skill`Bind Undead Elbow Macaroni`);
+        outfit(get("instant_stickKnifeOutfit"));
```

Comment:
I would add `|| have($familiar'Disembodied Hand') || my_baseStat($stat'Mysticality') < 250` to the completion condition, and do `if (myThrall() !== $thrall'Undead Elbow Macaroni' && my_baseStat($stat'Muscle') < 250)` to prior to summoning the thrall (there apparently are many high shiny users who appear to be hitting >= level 21 powerleveling, and they could very well have enough Muscle to equip the stick knife without the macaroni)